docs/source/markdown/podman-save.1.md

```diff
@@ -35,8 +35,9 @@ Write to a file, default is STDOUT
 
 #### **--format**=*format*
 
-Save image to **oci-archive, oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type)
+Save image to **docker-archive**, **oci-archive** (see `containers-transports(5)`), **oci-dir** (`dir` transport with oci manifest type), or **docker-dir** (`dir` transport with v2s2 manifest type).
```

Comment:
AFAICS `oci-dir` is `c/image/oci/layout`, not `c/image/directory`.

(BTW `AutocompleteImageSaveFormat` seems to be missing one of the options.)

Reply:
Oh, I need to track that down.  It could be a regression from libimage.

Reply:
OK. This is not a regression but a surprise to me. I'll update accordingly.